src/gwt/src/org/rstudio/studio/client/common/shell/ShellWidget.java

```diff
@@ -183,6 +193,34 @@ public void onRenderFinished(RenderFinishedEvent event)
          }
       });
 
+      Event.addNativePreviewHandler(new NativePreviewHandler()
```

Comment:
This is kind of a gross hack to ensure that Ctrl + F and Cmd + F use the Console search when the Console has focus. (It needs to be a native preview event so that it can have a chance to run before the shortcut manager handles it.)

Is this reasonable?